src/v1.js

```diff
@@ -373,7 +373,7 @@ api.declare({
   var payload = req.body;
   var hook = await this.Hook.load({hookGroupId, hookId}, true);
   var error = null;
-  const ajv = new Ajv({format: 'full', verbose: true, allErrors: true});
+  const ajv = new Ajv();
```

Comment:
Why this change?

Reply:
because I was testing, but verbose and allErrors are really important to "understand" what are doing the schemas fail, and of course format in cases that data, or hours are being used.